deno.json

```diff
@@ -1,9 +1,17 @@
 {
+  "name": "@cmd-johnson/deno-oauth2-client",
```

Comment:
Happy with this being the name of the package?

Reply:
Is there anything specific to Deno in the implementation of this package? If not I suppose the `deno` prefix could just be dropped, JSR is a general purpose registry.

Reply:
The deno.land package does not include the prefix https://deno.land/x/oauth2_client@v1.0.2

Reply:
Good point. Removed `deno-`.